server/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "type": "module",
   "scripts": {
-    "test": "NODE_ENV=test mocha ./test/* --exit",
+    "test": "NODE_ENV=test mocha \"test/**/*.test.js\" --exit",
```

Comment:
Curiosity: why this change?

Reply:
I don't know why, but the first one doesn't fully browse the tests directory, so I change that to run all tests, even those in sub-subfolders.